deploy/custom_resources/custom_authorizer/custom_authorizer_lambda.py

```diff
@@ -19,18 +20,25 @@
 
 def lambda_handler(incoming_event, context):
     # Get the Token which is sent in the Authorization Header
+    logger.debug(incoming_event)
```

Comment:
this will print the JWT token, are we using proper log-level to protect prod deployments?

Reply:
good call out - adding log_level setting on custom Authorizer Function in  `/deploy/stacks/lambda_api.py`